sys-utils/libcmt/tools/prepare-ffi.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+sed \
+	-e "/#if\s/,/#endif/d" \
```

Comment:
Would be nice to try also stripping comments, to save cycles on libffi parsing and speed up startup time.

Reply:
Doesn't make a difference if users use the method from the docs.
It only parses the headers during the build of `.so` python module.

Reply:
I see, doesn't make difference for Python its build step should take care of this. But for other languages,I tried (MicroPython, LuaJIT, Lua, QuickJS) they did not have build step, they parse and import on the fly. I would have to strip myself in case of them.

Reply:
Fair enough, added.